content/en/docs/languages/ruby/libraries.md

```diff
@@ -98,6 +114,33 @@ OpenTelemetry::SDK.configure do |c|
 end
 ```
 
+#### Configuring specific instrumentation libraries with environment variable
+
+You can also define the option for specific instrumentation libraries using
+environment variables. By convention, the environment variable will be the name
+of the instrumentation, with uppercase, `::` replaced by underscores,
```

Comment:
This matches what was used in the "Overriding configuration for specific instrumentation libraries with environment variable" section:
```suggestion
of the instrumentation, upcased with `::` replaced by underscores,
```